charts/datadog/values.yaml

```diff
@@ -1226,6 +1270,17 @@ clusterChecksRunner:
   #   - name: <ENV_VAR_NAME>
   #     value: <ENV_VAR_VALUE>
 
+  # clusterChecksRunner.envFrom -- Set environment variables for the Cluster Checks Runner directly from configMaps and/or secrets
+  ## envFrom to pass configmaps or secrets as environment
+  ## ref: https://github.com/DataDog/datadog-agent/tree/main/Dockerfiles/agent#environment-variables
+  envFrom: []
+  #   - configMapRef:
+  #       name: <CONFIGMAP_NAME>
+  #   - secretRef:
+  #       name: <SECRET_NAME>
+
+
+
```

Comment:
[Those extra blank lines are breaking the CI](https://github.com/DataDog/helm-charts/pull/393/checks?check_run_id=3700936167#step:5:48).
```suggestion
```